Makefile

```diff
@@ -92,7 +92,7 @@ endif
 export SQLALCHEMY_URL = "postgresql://$(PG_CREDENTIALS)@$(@_POSTGIS_IP):5432/pyramid_oereb_test"
 export PNG_ROOT_DIR = pyramid_oereb/standard/
 
-.coverage: $(PYTHON_VENV) $(TESTS_DROP_DB) $(TESTS_SETUP_DB) pyramid_oereb/standard/pyramid_oereb.yml .coveragerc $(shell find -name "*.py" -print)
+.coverage: $(PYTHON_VENV) $(TESTS_DROP_DB) $(TESTS_SETUP_DB) pyramid_oereb/standard/pyramid_oereb.yml .coveragerc $(shell find -name "*.py" -print | fgrep -v /.venv)
```

Comment:
Is this change for the point in your comment "exclude directory .venv as dependency in Makefile target .coverage"?
Why is this being excluded, in fact?

Reply:
Somehow, running `make clean-all checks` the target for `.coverage` failed because of a Python file of an installed package, that has been registered by the `find` call, but did not exist anymore.

My theory:
Running make, it first evaluates the dependencies between the single targets, also calling `find` in this case. This generates a list with all Python files of the installed packages at each start of a run. Then the target `clean-all` removes all this packages. The packages are installed  again as a dependency of the `checks` target, but maybe some versions of dependent packages could have changed, which may result in different Python files as registered at start. This leads to errors, like
```
No rule to make target '.venv/lib/python3.6/site-packages/some-package/some-file.py', needed by '.coverage'.  Stop.
```

That is why I think, the folder `.venv` has to be excluded.

Reply:
A proof would be if `make clean-all` followed by a separate `make checks` produces the same result as @kdeininger 's  change. Since the problem is coming up in really rare situations (like changing sub-sub package dependencies) we probably are not able to reproduce the case easily. Anyway... Excluding `.venv` directory from coverage report generation looks like a good idea.